cdp_scrapers/README.md

```diff
@@ -73,41 +83,48 @@ votes = reduced_list(
 
 ## Legistar -> CDP Ingestion
 
-Some collection of notes from development phase on what Legistar API fields to 
-use for what `IngestionModel` fields.
+Here are some notes on what Legistar API fields to use for `IngestionModel` fields.
 
-- For example "Public Comment" isn't a matter, its a minutes item. Minutes items 
+- "Public Comment" isn't a matter, it is a minutes item. Minutes items 
 are: "something that happened or was discussed during the meeting" while matters 
 are usually the "things that were discussed" which is why minutes can link to 
 matters. "Approval of Agenda" is a minutes item and links to the Matter "Agenda 
-for 2021-07-28" for example.
+for 2021-07-28", for example.
 
 ## Instance-specific scraping
 
 Base `LegistarScraper` was written such that ideally, an installation needs to 
-define a derived class with just `__init__()` defined where Legistar client name 
-is provided by calling `super().__init__("my_city")`, as in the above example.
+define a derived class with just a couple of methods implemented.
+
+```python
+class MyMunicipalityScraper(LegistarScraper):
+    def __init__(self):
+        """
+        municipality-specific implementation of LegistarScraper
+        """
+        super().__init__("municipality_legistar_id")
+
+    def get_time_zone(self) -> str:
+        return "America/Los_Angeles"
+```
```

Comment:
When I make my PR to add the dependency and installation management of scrapers to the repo, I may change the `get_time_zone` to be a param of the scraper init. Thoughts?

Reply:
Yes. I like the "required information should be provided in the constructor" line of thinking.